### PR TITLE
Update mrsim-app to updated dependencies

### PR DESCRIPTION
**Pull request by deepanshs**

### CODE REVIEW

File: app/assets/example_link.json
Comment (deepanshs) [inline app/assets/example_link.json:1]: @mgiammar Can we use the `https` links over `http`?

Comment (mgiammar) [reply app/assets/example_link.json:1]: Yes. I copied the links directly from the ssnmr.org website; did not realize the default to `http` links

File: app/custom_widgets.py [hunk 1]
```diff
@@ -121,7 +121,7 @@ def custom_slider(label="", return_function=None, **kwargs):
                 [label, dbc.FormText(id=id_label)],
                 className="d-flex justify-content-between",
             ),
-            dcc.Slider(**kwargs, className="slider-custom"),
+            dcc.Slider(**kwargs, class_name="slider-custom"),
```
Comment (deepanshs) [inline app/custom_widgets.py]: dcc.Slider has no attribute `class_name`. it's `className`

Comment (deepanshs) [reply app/custom_widgets.py]: fixed

File: app/assets/example_link.json
Comment (deepanshs) [reply app/assets/example_link.json:1]: fixed

**Pull request merged**